build.savant

```diff
@@ -64,10 +65,10 @@ groovyTestNG.settings.groovyVersion = "4.0"
 groovyTestNG.settings.javaVersion = "17"
 idea.settings.imlFile = "dependency-plugin.iml"
 idea.settings.moduleMap = [
-    "org.savantbuild:savant-core:2.0.0-RC.7"                  : "savant-core",
-    "org.savantbuild:savant-dependency-management:2.0.0-RC.7" : "savant-dependency-management",
-    "org.savantbuild:savant-io:2.0.0-RC.7"                    : "savant-io",
-    "org.savantbuild:savant-utils:2.0.0-RC.7"                 : "savant-utils"
+    ("org.savantbuild:savant-core:${savantVersion}".toString())                 : "savant-core",
+    ("org.savantbuild:savant-dependency-management:${savantVersion}".toString()): "savant-dependency-management",
+    ("org.savantbuild:savant-io:${savantVersion}".toString())                   : "savant-io",
+    ("org.savantbuild:savant-utils:${savantVersion}".toString())                : "savant-utils"
```

Comment:
Is this required? 

We use this in `fusionauth-app` w/out using `.toString()`. 

```
idea.settings.moduleMap = [
    "io.fusionauth:fusionauth-plugin-api:${fusionauthVersion}": "fusionauth-plugin-api"
]
```

Reply:
Reset this back to the method without the toStrings